socrates/lib/notifications/index.js

```diff
@@ -34,30 +38,34 @@ module.exports = {
   newParticipant: function (memberID, bookingdetails) {
     memberstore.getMemberForId(memberID, function (err, member) {
       if (err || !member) { return logger.error(err); }
-      var renderingOptions = {
-        pretty: true,
-        member: member,
-        url: conf.get('publicUrlPrefix'),
-        bookingdetails: bookingdetails
-      };
+      var options = renderingOptions(member);
+      options.bookingdetails = bookingdetails;
       var filename = path.join(__dirname, 'jade/registrationConfirmation.jade');
       var receivers = [member.email()];
-      notifications._sendMail(receivers, 'SoCraTes Registration Confirmation', jade.renderFile(filename, renderingOptions));
+      notifications._sendMail(receivers, 'SoCraTes Registration Confirmation', jade.renderFile(filename, options));
+      memberstore.allMembers(function (err, members) {
```

Comment:
Ist das nicht etwas arg, über 1200 Members in den Speicher zu laden, nur um 2 Mailadressen zu bekommen? Warum nicht für jede ID aus der Superuser-Liste den entsprechenden Member holen?